logparser/logs/events/events.py

```diff
@@ -137,6 +137,17 @@ def on_msg_size_reduced(match, state, logger):
                (transport, actual))
 
 
+def on_set_default_initial_peers(match, state, logger):
+    """Default initial peers depending on the enabled transports."""
+    if "udpv4" not in match[0]:
+        logger.cfg("Builtin UDPv4 is not enabled in the architecture", 5)
```

Comment:
LogParser hasn't a level 5 of verbosity yet. I don't see any reason to hide this message so much. I would recommend to set it to 2.